packages/core/status/core-status-common/src/service_status.ts

```diff
@@ -37,6 +37,11 @@ export interface ServiceStatus<Meta extends Record<string, any> | unknown = unkn
    * machine-readable information about the service status. May include status information for underlying features.
    */
   meta?: Meta;
+
+  /**
+   * Whether this status is reported by the service/plugin itself, or it is inferred from the status of other services that this service depends on
+   */
+  isReportedStatus?: boolean;
```

Comment:
From the status point of view, there are 2 types of plugins:
* Those that register a `status$` Observable and proactively report their status (there are 4 of them RN).
* Those that do NOT register a `status$` Observable, and their status is inferred from the status of their dependencies (plugin dependency tree) + status of core services: (elasticsearch client, saved objects).

This flag is basically telling us whether the status is directly reported or it is inferred.

From a logging perspective, the `(and 48 more)` refers to 48 other plugins that have changed status because of their dependencies, but they don't have a "self" status themselves (thus, not much value logging them).

Hope that answers your [other question](https://github.com/elastic/kibana/pull/168207#pullrequestreview-1667544494) too.

Reply:
Thinking about it, plugin owners should not need to worry about this property, it's managed internally by us, will check if I can extend the interface with an internal one.